vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureStoryReporter.java

```diff
@@ -441,6 +442,23 @@ public void onAttachmentPublish(AttachmentPublishEvent event)
         lifecycle.addAttachment(attachment.getTitle(), attachment.getContentType(), null, attachment.getContent());
     }
 
+    @Subscribe
+    public void onLinkPublish(LinkPublishEvent event)
+    {
+        lifecycle.updateTestCase(getRootStepId(), result ->
+        {
+            boolean exists = result.getLinks().stream()
```

Comment:
why is it needed?

Reply:
to avoid multiple links received from GivenStories scenarios at Scenario level